app/konnect/dev-portal/audit-logging/replay-job.md

```diff
@@ -0,0 +1,126 @@
+    --header "Content-Type: application/json" \
+    --header "Authorization: Bearer <personal-access-token>" \
+    --data '{
+        "start_at": "2023-03-27T20:00:00Z",
+        "end_at": "2023-03-27T20:00:00Z"
+    }'
+```
+
+Be sure to replace the PAT token and the following placeholder values:
+* `{region}.api.konghq.com`: The region your portal is located in. Can be `us`, `ap`, or `eu`.
+* `{portalId}`: The ID of the Dev Portal with your webhook.
+* `start_at` and `end_at`: Specify the timeframe for which you want to receive audit log events. `start_at` must be no more than seven days ago.
+
+If the request is successful, you will receive a `202` response code and a response body containing the replay job details: 
+
+```json
+{
+    "start_at":"2023-03-27T20:00:00Z",
+    "end_at":"2023-03-27T20:00:00Z",
+    "updated_at":"2023-03-31T11:34:18Z",
+    "status":"accepted"
+}
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## View replay job
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+1. In {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal), click the Dev Portal you want to view the replay job for.
+1. Click **Settings** in the sidebar, then click the **Audit Logs** tab.
+1. Click the **Replay** tab.
+1. Check the status table below the configuration field.
+
+{% endnavtab %}
+{% navtab API %}
+The {{site.konnect_short_name}} API uses [Personal Access Token (PAT)](/konnect/api/#authentication) authentication. You can obtain your PAT from the [personal access token page](https://cloud.konghq.com/global/account/tokens). The PAT must be passed in the `Authorization` header of all requests.
+
+You can view the audit log replay job in a given region by issuing a GET request to the `audit-log-replay-job` endpoint:
+
+```sh
+curl -i -X GET https://{region}.api.konghq.com/v2/portals/{portalId}/audit-log-replay-job \
+    --header "Authorization: Bearer TOKEN"
+```
+
+Be sure to replace the PAT token and the following placeholder values:
+* `{region}.api.konghq.com`: The region your portal is located in. Can be `us`, `ap`, or `eu`.
+* `{portalId}`: The ID of the Dev Portal with your webhook.
+
+You will receive a `200` response code and the job details:
+
+```json
+{
+    "start_at":"2023-03-27T20:00:00Z",
+    "end_at":"2023-03-27T20:00:00Z",
+    "updated_at":"2023-03-31T11:34:18Z",
+    "status":"accepted"
+}
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+## Replay job status
+
+A replay job can be in one of the following statuses:
+
+* `unconfigured`: Initial state. The job has not been set up.
+* `accepted`: The job has been accepted for scheduling.
+* `pending`: The job has been scheduled.
+* `running`: The job is in progress.
+* `completed`: The job has finished with no errors.
+* `failed`: The job has failed.
+
+When a replay job is `running`, a request to update the job will return a `409` response code until it has completed or failed.
```

Comment:
If you choose not to use a table, I would put this as a secondary bullet under `running`